Sources/AWSSDKSwiftCore/MetaDataService.swift

```diff
@@ -23,7 +23,9 @@ import class  Foundation.ProcessInfo
 /// errors returned by metadata service
 enum MetaDataServiceError: Error {
     case missingRequiredParam(String)
+    case couldNotGetInstanceSessionKey
```

Comment:
Currently not used

Reply:
you caught me, mid development here. Thought I was done yesterday but realised this morning there was a issue with running containers on EC2 and this setup